### PR TITLE
[BUU] Activate admin_style_v3 for 75% of users

### LINKED ISSUE #12756: [BUU] Activate admin_style_v3 for 75% of users
⚠ Please use clockify code https://github.com/openfoodfoundation/openfoodnetwork/issues/7198 Back Office Uplift (Product) while reviewing or working on this task.

**TO BE RELEASED On August 20th**
(see [schedule](https://community.openfoodnetwork.org/t/backoffice-ui-product-list-uplift-rollout/2921/35))

## Description
**- As a:** super admin
**- On page:** `/admin/feature-toggle/features`
**- I want to be able to do:**  See that on top of the admin and new users groups, the toggle is also activated for 75% of users

## Acceptance Criteria & Tests

1. Before the PR only admins + new user + 75% + dedicated actors eventually should have had access
2. After: same as 1 + 75% of users instead of 50

We will trust flipper to work on this one, we won't check if it is really 75%

### PR DESCRIPTION
**Pull request by EdwardLi-coder**
#### What? Why?

- Closes  #12756

I have adjusted the admin_style_v3 to 75.But I'm not sure if this change is suitable.
If the team has any suggestions.I'm happy to make further adjustments.

#### What should we test?

- Visit ... page.
- 

#### Release notes

Changelog Category (reviewers may add a label for the release notes):

- [ ] User facing changes
- [ ] API changes (V0, V1, DFC or Webhook)
- [ ] Technical changes only
- [x] Feature toggled

The title of the pull request will be included in the release notes.

### CODE REVIEW

**Review by rioug — APPROVED** [review]
Looks good :+1: 

@dacook do we need to test this ?

File: lib/open_food_network/feature_toggle.rb [hunk 1]
```diff
@@ -51,7 +51,7 @@ module FeatureToggle
       # Copy features here that were activated in a migration so that new
       # instances, development and test environments have the feature active.
       "admin_style_v3" => <<~DESC,
-        Test the work-in-progress design updates.
+        Test the work-in-progress design updates. Activated for admins, new users, and 75% of all users.
```
Comment (dacook) [inline lib/open_food_network/feature_toggle.rb:54]: Nice idea to keep the description updated 👍 

However, these details are not hard coded and can be manipulated in the admin interface according to specific needs (see below). So they could be wrong, and cause confusion. Can you please remove this change?

See http://localhost:3000/admin/feature-toggle/features/admin_style_v3
Screenshot 2024-08-12 at 5 42 51 pm

**Review by dacook — CHANGES_REQUESTED** [review]
Thank you! There's just one thing to change first.

File: lib/open_food_network/feature_toggle.rb (hunk 1 above)
Comment (dacook) [reply lib/open_food_network/feature_toggle.rb:54]: I just realised I can do that easily now.
```suggestion
        Test the work-in-progress design updates.
```

**Comment by dacook** [comment]
FYI @rioug, I've changed the label as we have decided now to consider any BUU changes as user-facing: https://openfoodnetwork.slack.com/archives/C01T75H6G0Z/p1722935037211829

Regarding testing, I don't think it's necessary because I can see it's exactly the same method used in the previous migrations.

So we are good to go. I have a feeling this was meant to be included in the current release.. I will check in Slack.

**Review by dacook — APPROVED** [review]

**Comment by dacook** [comment]
Actually I just checked the issue, it's scheduled for the next one, so I will merge now.

> **TO BE RELEASED On August 20th**

**Pull request merged**